18F/identity-idp
# LG-6220: Add password confirm text content (IdV app)

**Why**: For parity with the existing screen, we should include all existing content which explains the reason for the user to enter their password.

**Testing Instructions:**

You will need to set the step as enabled in your local `config/application.yml`:

```yml
development:
  idv_api_enabled_steps: '["password_confirm","personal_key","personal_key_confirm"]'
```

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to password confirmation step
5. Observe additional content, and link which opens the [Login.gov Security Practices page](https://login.gov/policy/our-security-practices/) in a new tab

**Screenshots:**

Before|After
---|---
![Screen Shot 2022-05-18 at 9 55 28 AM](https://user-images.githubusercontent.com/1779930/169059075-7ebb0a5f-0418-49ef-ad09-b1865690b1aa.png)|![Screen Shot 2022-05-18 at 9 55 09 AM](https://user-images.githubusercontent.com/1779930/169059071-2c4ea731-43cd-4eb0-978b-189f049f83c9.png)
